include/memcached_cache.h

```diff
@@ -291,15 +292,29 @@ class MemcachedCache : public BaseHssCache
 {
 public:
   MemcachedCache(ImpuStore* local_store,
-                 const std::vector<ImpuStore*>& remote_stores) :
+                 const std::vector<ImpuStore*>& remote_stores,
+                 int num_threads,
+                 ExceptionHandler* exception_handler) :
     BaseHssCache(),
     _local_store(local_store),
-    _remote_stores(remote_stores)
+    _remote_stores(remote_stores),
+    _thread_pool(num_threads,
+                 exception_handler,
+                 exception_callback,
+                 0)
   {
+    _thread_pool.start();
   }
 
   virtual ~MemcachedCache()
   {
+    _thread_pool.stop();
+    _thread_pool.join();
+  }
+
+  // Dummy exception handler callback for the thread pool
```

Comment:
Do we not want a real exception callback? Something that logs out something useful?

Reply:
This matches what we do in the cache processor. We don't need to do anything if we've hit an exception, so I think this is fine.